### ISSUE TITLE
Slicerator 0.9.5

### ISSUE THREAD

**Issue by caspervdw** [soft-matter/slicerator#14]
Is there anything we should do before the release?

**Comment by danielballan** [comment]
I think this is good to know. Interested in rolling out this release
yourself? I think I need to give you access to or PyPI account.

**Comment by danielballan** [comment]
*good to go, not good to know.

**Comment by caspervdw** [comment]
I could do the PyPI upload, no problem. For conda I am not sure how to do that because I've never done that before.

**Comment by caspervdw** [comment]
@danielballan  If you want me to make the release could you give me access to the PyPi account?

**Comment by caspervdw** [comment]
Tagged and uploaded to pypi!

**Comment by danielballan** [comment]
Nicely done. Tom and I are in hair-on-fire mode this week at BNL, but I'll get to conda packages ASAP.

**Comment by caspervdw** [comment]
Not so nice ... pims build is failing on Py2.7 because of a versioneer error. I redid the release and made sure that I got it right but still no success on the travis build.

https://travis-ci.org/soft-matter/pims/jobs/100895708

**Comment by caspervdw** [comment]
I looked at it once more and the most probable thing is that it is an issue between windows and versioneer. I used the exact recipe from trackpy and for some reason, versioneer does write the correct `_slicerator_version.py`, but when travis tries to install it, versioneer cannot find the version and returns `0+unknown`, only at Py2.7.

Just to quickly solve things (fresh slicerator/pims/trackpy installations on Py2.7 may fail because of this issue), could any of you send the release out in the way you always do it so that versioneer works and we are back online?

We should tag 0.9.7 already.

**Comment by caspervdw** [comment]
I narrowed down the issue: because I am working on windows, all the files have CR/LF line endings. Python is OK with that, however versioneer expects LF line endings.

I raised an issue with versioneer, for now I manually edit the `_slicerator_version.py` file to LF line endings and proceed with the release.

**Comment by danielballan** [comment]
OK, sounds good.

**Comment by danielballan** [comment]
Did you want me to do it (on my non-Windows machine, I mean) to be safe, or are you confident you've got it figured out? An RC is probably advisable either way.

**Comment by caspervdw** [comment]
It took me another 5 rc's but it is working again :smiley: Sorry about the version numbers, I should have started with RC's right away.

In the end I built the source distribution tree with `setup.py sdist --keep-temp`, then converted each file in the tree from CRLF to LF with `io.open`, and then used the `distutils.archive_util.make_archive` method to create the tarballs. I kept the script so for next time it will be easy.

I cleaned up the faulty releases from PyPi so it jumps from 0.9.4 to 0.9.7. The tags are partly still in the git master, shall I remove them?

Closing this issue.
